rakelib/kokoro/kokoro.rb

```diff
@@ -193,10 +193,10 @@ def job_info gem = nil
   end
 
   def load_env_vars
-    service_account = "#{ENV['KOKORO_GFILE_DIR']}/service-account.json"
+    service_account = "#{ENV['KOKORO_GFILE_DIR']}/yoshi-ruby-service-account.json"
     raise "#{service_account} is not a file" unless File.file? service_account
-    ENV["GOOGLE_APPLICATION_CREDENTIALS"] = service_account
-    filename = "#{ENV['KOKORO_GFILE_DIR']}/env_vars.json"
+    ENV["GOOGLE_CLOUD_CREDENTIALS"] = service_account
+    filename = "#{ENV['KOKORO_GFILE_DIR']}/ruby_env_vars.json"
```

Comment:
Are we sure that these changes will work across the other kokoro jobs (including release jobs)?

Reply:
ruby_env_vars.json is copied from env_vars.json with GOOGLE_CLOUD_PROJECT set. I'll rollback the GOOGLE_APPLICATION_CREDENTIALS -> GOOGLE_CLOUD_CREDENTIALS change just to be safe.

Reply:
That's reasonable for now. Something we should think about longer term is if we can determine a way to test these kinds of changes other than just submitting them and seeing what breaks.